mono/metadata/profiler.h

```diff
@@ -196,6 +200,99 @@ typedef MonoProfilerCallInstrumentationFlags (*MonoProfilerCallInstrumentationFi
  */
 MONO_API void mono_profiler_set_call_instrumentation_filter_callback (MonoProfilerHandle handle, MonoProfilerCallInstrumentationFilterCallback cb);
 
+/*
+ * Enables support for retrieving stack frame data from a call context. At the
```

Comment:
It just works, except for the special case regarding LLVM-only that's mentioned in the comment.